packages/patterns-react/src/patterns/PictogramArray/README.md

```diff
@@ -0,0 +1,78 @@
+# Content array with pictograms
+
+> The Content array with pictograms pattern is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { PictogramArray } from '@carbon/ibmdotcom-patterns-react';
+import { Desktop, Touch, Pattern } from '@carbon/pictograms-react';
+import 'yourapplication.scss';
+import '@carbon/ibmdotcom-styles/scss/patterns/pictogramarray/index.scss';
+
+function App() {
+  return;
```

Comment:
@RaphaelAmadeu minor thing, might help in the example to define `title` and `contentGroup`.

Reply:
@jeffchew I've included this in the latest version. Thanks!

Reply:
I meant something like this:

```javascript
import { Pattern } from '@carbon/pictograms-react';

function App() {
  const title = 'Lorem Ipsum';
  const contentGroup = {[ 
    {
      title: 'Lorem', 
      copy: '', 
      link: {
        href: 'https://www.ibm.com', 
        text: 'IBM', 
        target: '_self'
      }, 
      pictogram: Pattern
    }
  ]};
  <PictogramArray title={title} contentGroup={contentGroup} />;
 };
}
```

Also, that `return;` before the `<PictogramArray>` doesn't seem to make sense.